src/styles/Contact.css

```diff
@@ -3,4 +3,85 @@
   height: 100vh;
   background-image: url('../img/img-bg-dark.jpg');
   background-size: 100% 100%;
-}
\ No newline at end of file
+  position: relative;
+  text-align: center;
+  justify-content: center;
```

Comment:
Contact.css: Code is clean and well-organized for future additions/amendments. `vertical-align: middle;` can be moved from `#Header`, `#Chat`, `#Help`, `#Follow`, and `#Copyright` to `.Contact`.